core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -605,6 +603,22 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  def unZip(inFile: File, untarDir: File): Unit = {
+    val untarCommand = new StringBuffer
```

Comment:
No need for a StringBuffer (StringBuilder is preferred anyway), just use an interpolated string

Reply:
Yes, that's a good suggestion. I'm just worried about concurrency.

Reply:
There will be no concurrency issue here